components/common/Banner/AppJamBanner/AppJamBanner.tsx

```diff
@@ -0,0 +1,115 @@
+import styled from '@emotion/styled';
+import Head from 'next/head';
+import React, { FC } from 'react';
+
+import BannerLeft from '@/components/common/Banner/AppJamBanner/assets/banner-left.svg';
+import BannerRight from '@/components/common/Banner/AppJamBanner/assets/banner-right.svg';
+import BannerRightMobile from '@/components/common/Banner/AppJamBanner/assets/mobile-banner-right.svg';
+import Responsive from '@/components/common/Responsive';
+import Text from '@/components/common/Text';
+import { MOBILE_MEDIA_QUERY } from '@/styles/mediaQuery';
+
+const APPJAM_LINK = 'https://www.notion.so/go-sopt/APP-JAM-0303e38562074483bfa9893d7df23147';
+
+const AppJamBanner: FC = () => {
+  return (
+    <>
+      <Head>
+        <link
+          rel='preload'
+          href='https://cdn.jsdelivr.net/gh/projectnoonnu/noonfonts_2212@1.0/KIMM_Bold.woff2'
+          as='font'
+          type='font/woff2'
+          crossOrigin=''
+        />
+      </Head>
+      <Responsive only='desktop'>
+        <StyledBanner>
+          <LeftImage />
+          <RightImage />
+          <TextWrapper>
+            <Text typography='SUIT_24_B'>
+              🗯&nbsp;
+              <StyledSOPT>GO SOPT</StyledSOPT> 기획 자료를 공개합니다&nbsp;🗯
+            </Text>
+            <StyledLink href={APPJAM_LINK} target='_blank' rel='noreferer noopener'>{`바로 보기 >`}</StyledLink>
+          </TextWrapper>
+        </StyledBanner>
+      </Responsive>
+      <Responsive only='mobile'>
+        <a href={APPJAM_LINK} target='_blank' rel='noreferer noopener noreferrer'>
```

Comment:
```suggestion
        <a href={APPJAM_LINK} target='_blank' rel='noopener noreferrer'>
```

noreferer는 필요 없는 거 아닌가용??

Reply:
어엇 왜 두개 들어갔죠.. 지우겠슴다 감사합니당